airbyte-webapp/src/components/StreamTestingPanel/ResultDisplay.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from "classnames";
+
+import { Text } from "components/ui/Text";
+
+import { StreamRead } from "core/request/ConnectorBuilderClient";
+
+import styles from "./ResultDisplay.module.scss";
+
+interface ResultDisplayProps {
+  streamRead: StreamRead;
+}
+
+export const ResultDisplay: React.FC<ResultDisplayProps> = ({ streamRead }) => {
```

Comment:
This component is not at all finalized -- this was just a quick and dirty implementation to display all of the contents of the `StreamRead` object in order to show that the data was successfully retrieved and give a hint at what it will look like.

This component will be the focus of the next PR, in which I will work on bringing it closer to the [design](https://www.figma.com/proto/F1ljvrJerLoVbNvtejBWid/LOW-CODE-CONNECTOR-BUILDER?node-id=1926%3A72640&scaling=min-zoom&page-id=1926%3A72574&starting-point-node-id=1926%3A72575)